pkg/server/server_test.go

```diff
@@ -2032,3 +2032,88 @@ func TestServerMaintenanceMode(t *testing.T) {
 	err = s.StreamExecAll(nil)
 	require.Contains(t, err.Error(), ErrNotAllowedInMaintenanceMode.Error())
 }
+
+func TestServerDatabaseTruncate(t *testing.T) {
+	dir := t.TempDir()
+	opts := DefaultOptions().WithDir(dir)
+
+	s := DefaultServer()
+	s.WithOptions(opts)
+
+	s.Initialize()
+
+	r := &schema.LoginRequest{
+		User:     []byte(auth.SysAdminUsername),
+		Password: []byte(auth.SysAdminPassword),
+	}
+
+	ctx := context.Background()
+	lr, err := s.Login(ctx, r)
```

Comment:
new code should rely on session-based authentication as login/logout is deprecated and we should gradually remove all the code using it

Reply:
fixed